blocks/ticket/ticket.ts

```diff
@@ -356,11 +356,17 @@ export class Ticket {
               cbk.getVariable(attachmentVar.variable) ?? "[]"
             );
             for (const uploadedFile of uploadedFiles) {
-              attachmentPayload.push({
-                fileName: uploadedFile.fileName,
-                s3Id: uploadedFile.fileKey,
-                ticketId: ticket.id,
-              });
+              // filter out pdf version of docx file (not display in files tab)
+              if (
+                !uploadedFile.reportName ||
+                uploadedFile.fileName === uploadedFile.reportName
+              ) {
```

Comment:
@sschye21 The if condition here might make others scratch their head in the future, I think we are better off filtering based on filename if the intention is just to filter out pdf eg.
```
if(!uploadedFile.fileName.match(/\.pdf$/)) {
  //...
}
```